understanding/accessible-authentication-minimum.html

```diff
@@ -336,97 +252,69 @@ <h3>Object Recognition</h3>
                   </ul>
                   
                   
-                  <p>None of these systems are 100% effective. However, they may reduce the likelihood
-                     of a CAPTCHA being displayed.
+                  <p>これらのシステムはいずれも 100％有効ではない。しかし、CAPTCHA が表示される可能性を減らすことはできる。
                   </p>
                   
                </section>
                <section id="personal-content">
                   
-                  <h3>Personal Content</h3>
+                  <h3>個人特有のコンテンツ</h3>
                   
-                  <p>Personal content is sometimes used as a second factor for authentication. For example,
-                     as part of account creation the user would upload a picture, and when logging in they
-                     would be asked to select that picture from several possible alternatives. Care must
-                     be taken to provide adequate security in this case, since non-legitimate users might
-                     be able to guess the correct personal content when presented with a choice.
+                  <p>個人特有のコンテンツは、認証の第 2 要素として使用されることがある。例えば、アカウント作成の一環として利用者が写真をアップロードし、ログイン時にいくつかの可能な選択肢からその写真を選択するように求められる。この場合、非正規の利用者が選択肢を提示されたときに正しい個人特有なコンテンツを推測できる可能性があるため、適切なセキュリティを提供するように注意しなければならない。
                   </p>
                   
                   
-                  <p>Text-based personal content does not qualify for this exception as it relies on recall
-                     (rather than recognition), and transcription (rather than selecting an item). Whilst
-                     picture-based personal content will still be a barrier for some people, text based
-                     versions tend to be a much larger barrier.
+                  <p>テキストベースの個人特有のコンテンツは、(認識ではなく) 記憶及び (項目の選択ではなく) 転記に依存しているため、この例外の対象にはならない。写真ベースの個人特有のコンテンツが依然として障壁となる人もいるが、テキストベースのバージョンははるかに大きな障壁となる傾向がある。
                   </p>
                   
                </section>
                <section id="hiding-content">
                   
-                  <h3>Hiding characters</h3>
+                  <h3>文字を隠す</h3>
                   
-                  <p>Another factor that can contribute to cognitive load is hiding characters when typing.
-                     Although this criterion requires that users do not have to type in (transcribe) a
-                     password, there are scenarios where that is necessary such as creating a password
-                     to be saved by a password manager. Providing a feature to optionally show a password
-                     can improve the chance of success for some people with cognitive disabilities or those
-                     who have difficulties with accurately typing.
+                  <p>認知的負荷につながる可能性のあるもう一つの要因は、入力時に文字が隠れることである。この基準では、利用者がパスワードを入力 (転記) する必要がないことが求められているが、パスワードマネージャーで保存するパスワードを作成するなど、パスワードが必要となるシナリオもある。オプションでパスワードを表示する機能を提供することで、認知障害のある人又は正確に入力するのが困難な人の成功の可能性が高まる。
```

Comment:
typing は「タイピング」で良いかも。上のほうでは「転記させること、たとえば文字をタイピングさせるなど」としています。

here are scenarios where that is necessary の that を「パスワード」と訳しているように思いますが、ここでは "to type in (transcribe) a  password" 全体を指しているように思います。単に「それ」としてしまっても支障はないように思いますが。

```suggestion
                  <p>認知的負荷につながる可能性のあるもう一つの要因は、タイピングの際に文字が隠れることである。この基準では、利用者がパスワードをタイピング (転記) する必要がないことが求められているが、パスワードマネージャーで保存するパスワードを作成するなど、それが必要となるシナリオもある。オプションでパスワードを表示する機能を提供することで、認知障害のある人、又は正確なタイピングが困難な人の成功の可能性が高まる。
```